Monika After Story/game/script-topics.rpy

```diff
@@ -4558,8 +4558,8 @@ label monika_hotdogcontest:
     m 2e "And your health means the world to me."
     m 3f "I'm not saying you should become vegetarian like I am."
     m "But I wouldn't oppose if you did."
-    m "Or are you already?"
     menu:
+    m "Or are you already?"
```

Comment:
Okay again, this needs to be indented once. I'll just mark that you need to do changes everytime i see this instead of copypasta this comment.